tools/build_wheels.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import logging
+import subprocess
+import sys
+
+from _utils import _LogWrapper
+
+_logger = logging.getLogger(__file__)
+logging.basicConfig(level=logging.INFO)
+
+
+def build_argument_parser():
+    desc = "Build wheels for interpret-text"
+
+    parser = argparse.ArgumentParser(description=desc)
+    parser.add_argument("--version-filename",
+                        help="The file where the version will be stored.",
+                        required=True)
+
+    return parser
+
+
+def main(argv):
+    parser = build_argument_parser()
+    args = parser.parse_args(argv)
+
+    with _LogWrapper("installation of interpret-text"):
```

Comment:
nice setup with these!

Reply:
You can thank Roman for the `_LogWrapper` class.